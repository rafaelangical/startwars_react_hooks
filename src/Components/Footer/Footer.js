import React from "react";

export default function Footer({ onClick, text }) {
  return (
    <footer className="button-botttom">
      <button type="submit" onClick={onClick}>
        {text}
      </button>
    </footer>
  );
}
