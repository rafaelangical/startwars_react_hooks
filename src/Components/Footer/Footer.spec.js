import React from "react";
import renderer from "react-test-renderer";
import Footer from "./Footer";

it("renders correctly footer", () => {
  const tree = renderer.create(<Footer text="Carregar mais" />).toJSON();
  expect(tree).toMatchSnapshot();
});
