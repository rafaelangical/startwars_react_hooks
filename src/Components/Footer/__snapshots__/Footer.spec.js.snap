// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly footer 1`] = `
<footer
  className="button-botttom"
>
  <button
    type="submit"
  >
    Carregar mais
  </button>
</footer>
`;
